_pages/setup-wallets/paper.md

```diff
@@ -72,11 +71,13 @@ See [advanced section](paper-advanced) for more info.
 
 
 ## Export Public Key Info to Specter-Desktop
```

Comment:
Maybe we need two separate steps (one for QR and one for doing it via a file)?

Reply:
That's a good idea. I'll update it to have 2 seperate steps so users can choose their preferred way.